packages/nimble-components/src/toggle-button/template.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
I don't understand why it complains about the `??` operator (`Using 'LogicalExpression[operator='??']' is not allowed.`). I disabled the rule to get around it.

Reply:
This is our rule, added in #2025 .

Based on the [comment here](https://github.com/ni/nimble/issues/1843#issuecomment-2064663561), I think what you have here is OK though, because the binding isn't _only_ the null coalescing operator (you have `? :` too).

Can see if others want to avoid it entirely (vs. turning off the rule) or not.

Reply:
Should we try to upstream a change to FAST that includes `??` in the regex used to identify volatile bindings?

Reply:
Yea there are two issues in the comment Malcolm linked thtat should be reported and fixed upstream. One is adding support for `??` and the other is that aggregate bindings don't ever work with the regex to mark a binding volatile.

Reply:
I will keep a note to make those changes when I have nothing more valuable to do.

Reply:
Can you make that note an issue for reporting the bugs to FAST so we can track it?

Reply:
Created #2127